DemoApp/Shared/StreamChatWrapper.swift

```diff
@@ -117,10 +113,10 @@ extension StreamChatWrapper {
                 if let error = error {
                     log.warning("removing the device failed with an error \(error)")
                 }
+
+                client.logout(completion: completion)
```

Comment:
I do think that removing the device should be part of the logout call by default. With an option to disable it if the customers wants it. The reason is that it is easy to make this mistake. If the customer does not wait for the removeDevice call, and then logout, it can cause unexpected issues.